tests/Toolkit/HasI18nTest.php

```diff
@@ -6,6 +6,11 @@
 
 class HasI18nTest extends TestCase
 {
+	public function setUp(): void
+	{
+		$this->setUpI18nLoader();
```

Comment:
`Thought`: Wondering whether we should actually test with Kirby core translations. Those can change, breaking tests even without any code changes. Maybe we should rather populate `I18n::$translations[$locale] = [<some test strings>]` and unset the array again in `tearDown()`. This would isolate the tests as much as possible and also make it clearer what data we test on.